sktm/jenkins.py

```diff
@@ -82,13 +86,46 @@ def __get_build(self, job, buildid):
                 return build
             except Exception as e:
                 logging.warning("catch %s: %s" % (type(e), e))
-                logging.info("now sleep 60s and try again")
-                time.sleep(60)
+                logging.info("now sleep %ds and try again" % interval)
+                time.sleep(interval)
 
         logging.error("fail to get build after retry %d times" %
                       self.retry_cnt)
         raise e
 
+    def __get_job_prop(self, job, method, interval):
+        """
+        Get property of Jenkins job. Retry Jenkins self.retry_cnt times
+        in case of temporary network failures.
+
+        Args:
+            job:        Jenkins job.
+            method:     Job method, it can be:
+                        - get_build_ids
```

Comment:
I don't think naming the methods is needed, since in theory any method of the Job class can be used. Can you please change the docstring to reflect that?

Reply:
The docstring is updated, and use attr instead.